simpleConditional/test_sortThreeNumberAscending.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from sortThreeNumberAscending import sortThreeNumberAscending
+
+class TestSortThreeNumberAscending(unittest.TestCase):
+    def test_listOfThreeIntegerInput(self):
+        self.assertEqual(sortThreeNumberAscending([5, 4, 3]), [3, 4, 5])
```

Comment:
Should not be in order like this. What if your program have bug and it just reverse your list?

Reply:
Acknowledge.